src/parser/test/ParserTest.cpp

```diff
@@ -1098,4 +1098,25 @@ TEST(Parser, ConfigOperation) {
     }
 }
 
+TEST(Parser, Limit) {
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 1 OVER work | LIMIT 1";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 1 OVER work | LIMIT 1,2";
```

Comment:
Great. But what do you think about `offset`:
select * from table limit 2,1;  
select * from table limit 2 offset 1;